include/dali/core/geom/mat.h

```diff
@@ -265,7 +265,7 @@ struct mat {
 
   template <typename U>
   DALI_HOST_DEV
-  inline auto operator*(const vec<cols, U> &v) {
+  inline auto operator*(const vec<cols, U> &v) const {
```

Comment:
I think there should be a comment, what multiplication is it. Carthesian or scalar?

Reply:
Is this a serious question?